app/controllers/application_controller.rb

```diff
@@ -4,7 +4,13 @@ class ApplicationController < ActionController::Base
   private
     def set_current_user
       if session[:current_user]
-        @current_user = User.find(session[:current_user])
+        begin
+          @current_user = User.find(session[:current_user])
+        rescue
```

Comment:
In general, you want to avoid broadly matching `rescue` blocks because they could hide bugs that you would otherwise know about immediately. Unless this part of the code is non-critical (e.g., logging/telemetry/tracking), you'll want to supply a relatively specific exception class, such as:

```suggestion
        rescue ActiveRecord::RecordNotFound
```

In this case, a broadly matching `rescue` could conceal a bug where many/all users are unable to keep their session, and you wouldn't have anything logged for it.